2021/01/13 18:01:34 : orders_errors 1: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:34 : orders_errors 2: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:35 : orders_errors 3: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:35 : orders_errors 4: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:36 : orders_errors 5: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:37 : orders_errors 6: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:37 : orders_errors 7: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

2021/01/13 18:01:38 : orders_errors 8: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4376, in order_import
    'meta_value': int(get_current_time().time())
AttributeError: 'str' object has no attribute 'time'

